### ISSUE TITLE
Question: Does the scale size stay upon reboot?

### ISSUE THREAD

**Issue by TheDevster** [kibook/redm-scalesync#1]
If I set a player ped scale size, does it maintain through reboots and logoffs?

**Comment by kibook** [comment]
No, it does not. It's something that could maybe be added, though.

**Comment by TheDevster** [comment]
It's not something that is added to skin right? .. so I would probably have to create a value that is stored and then at "character load" check for that value and then call the setScale function?
